Dockerfile

```diff
@@ -0,0 +1,14 @@
+# base image
+FROM node
+
+#Work Directory in container
+WORKDIR /usr/app
+
+# COPY package.json and source code files
+COPY ./ /usr/app
+
+# Install dependencies
+RUN npm install
+
+# Set up a default command
+CMD [ "npm","run","dev" ]
```

Comment:
So is this docker container is created only for local development? Then I don't quite understand which problem it solves.

If you're working on the project locally, you need to have `node` and `npm` in order to develop this application (no IDE will work with the project without having node or npm installed). So any web-developer will already have all necessary dependencies and utilities installed in order to run this project locally. 

Because of that, Docker (at least in my company) is usually used to package application for deployment to production servers. Where required utilities and dependencies might not be present.

Reply:
If you have another approach to local development and you can tell about benefits of packaging application in Docker container for dev environment, then I will be happy to hear about it.

Reply:
Though, it might be beneficial for people who want to try to run this project locally for the first time without installing any dependencies and tools. Ok, then I will approve. But I propose to remove docker-compose because it is currently unnecessary.

Reply:
Hi ,
Thank you for considering my contribution
Yes this is not user perspective change. This is for contributor's convenience.  You are right about "it might be beneficial for people who want to try to run this project locally for the first time without installing any dependencies"

- It was my personal choice. I have been working with docker so I thought why not add 3 files to remote if someone in future consider it . They do not have to put extra efforts or have to do  less efforts
- Second thought was in case someone face dependency/Platform issue in future we do not have to change local env setup or play around dependency version

Reply:
@harsh8051 make sense. Sorry for asking you to remove docker-compose. I will add it back.